app/models/organization.rb

```diff
@@ -0,0 +1,56 @@
+# Organisations (schools etc.) have teachers and their own customized courses.
+
+class Organization < ActiveRecord::Base
+  validates :name,
+            presence: true,
+            length: { within: 2..40 },
+            uniqueness: true
+  validates :information, length: { maximum: 500 }
+  validates :slug,
+            presence: true,
+            length: { within: 2..20 },
+            format: {
+              with: /\A[a-z0-9\-_]+\z/,
+              message: 'must be lowercase alphanumeric and may contain underscores and hyphens'
+            },
+            uniqueness: true
+  validates :acceptance_pending, inclusion: [true, false]
+  validate :valid_slug?, on: :create
+
+  has_many :teacherships, dependent: :destroy
+  has_many :users, through: :teacherships
```

Comment:
Could we maybe call this field as teachers (instead of the teachers 'alias' hack)